apps/accessibility/src/accessibilityoca.js

```diff
@@ -4,7 +4,7 @@
  * @author Jan C. Borchardt <hey@jancborchardt.net>
  * @author John Molakvoæ <skjnldsv@protonmail.com>
  *
- * @license GNU AGPL version 3 or any later version
+ * @license GPL-3.0-or-later
```

Comment:
Isn't this wrong as Nextcloud is indeed AGPL, not GPL?

Reply:
good point!

Reply:
Even though, you can't change the licence like that.
```suggestion
 * @license AGPL-3.0-or-later
```

Reply:
The AGPL explicitly applies additional statements to ensure that e.g. hosting providers need to make changes to the source public as well: https://github.com/nextcloud/03a0f4c/blob/master/COPYING#L42-L48

An important addition for Nextcloud which is indeed offered a lot by hosting providers.

Reply:
in any case, this ticket is just about making the linter happy, not changing the license :sweat_smile: 

so fixing the identifier to say AGPL makes sense

Reply:
Reseted to AGPL